adagrams/game.py

```diff
@@ -1,13 +1,182 @@
 from random import randint
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
```

Comment:
This is nice work, using the python convention of using all caps to signal that these variables are not meant to be changed, constant variables. Something to consider when making constant variable, and variables in general, is the scope. The larger the scope the more functions and operations have access to them. This can create more chances for them to unintentionally changed. So, a question I tend to ask myself is "what needs access to the variable I am about to create?".

Also, to make the file seem a little less padded and cluttered, consider making multiple key/value pairs sit on a single line like so: 
```suggestion
LETTER_POOL = {
    "A":9,"B":2,"C":2,"D":4,
    "E":12,"F":2,"G":3,"H":2,
    "I":9,"J":1,"K":1,"L":4,"M":2,
    "N":6,"O":8,"P":2,"Q":1,
    "R":6,"S":4,"T":6,"U":4,
    "V":2,"W":2,"X":1,
    "Y":2,"Z":1
}
```

Reply:
Thanks! Next time, I will consider this!

Reply:
As far as formats go, I probably would prefer my format. I find it easier to read and check for possible inaccuracies that way.